### ISSUE TITLE
Plugin with id 'com.facebook.react' not found.

### ISSUE THREAD

**Issue by Asinox** [th3rdwave/react-native-safe-area-context#316]
Any idea how to fix this when `newArchEnabled=true` ?

`Plugin with id 'com.facebook.react' not found.`

Thank you!

**Comment by janicduplessis** [comment]
What version of react native are you using? The lastest version of this library only supports 0.70

**Comment by efstathiosntonas** [comment]
@janicduplessis just to be clear, version `4.4.1` will work only on rn >= 0.70.0?

**Comment by janicduplessis** [comment]
Yes for the new architecture there are too many changes currently to support older versions so for now this library will only support the latest version. I think you can stay on 4.3 for RN 0.69
